src/chunkserver/datastore/chunkserver_snapshot.cpp

```diff
@@ -143,23 +183,21 @@ CSErrorCode CSSnapshot::Open(bool createFile) {
     // The existence of snapshot files may be caused by the following conditions
     // getchunk succeeded, but failed later in stat or loadmetapage,
     // when the download is opened again;
-    if (createFile
-        && !lfs_->FileExists(snapshotPath)
-        && metaPage_.sn > 0) {
+    if (createFile && !lfs_->FileExists(snapshotPath) && metaPage_.sn > 0) {
         std::unique_ptr<char[]> buf(new char[pageSize_]);
         memset(buf.get(), 0, pageSize_);
         metaPage_.encode(buf.get());
         int ret = chunkFilePool_->GetFile(snapshotPath, buf.get());
         if (ret != 0) {
             LOG(ERROR) << "Error occured when create snapshot."
-                   << " filepath = " << snapshotPath;
+                       << " filepath = " << snapshotPath;
             return CSErrorCode::InternalError;
         }
     }
-    int rc = lfs_->Open(snapshotPath, O_RDWR|O_NOATIME|O_DSYNC);
+    int rc = lfs_->Open(snapshotPath, O_RDWR | O_NOATIME | O_DSYNC);
```

Comment:
Do not introduce code style changes unrelated to the issue

Reply:
ok, i will fix it